mypy/subtypes.py

```diff
@@ -2208,7 +2208,8 @@ def infer_variance(info: TypeInfo, i: int) -> bool:
                     settable = False
 
             # TODO: handle settable properties with setter type different from getter.
-            typ = find_member(member, self_type, self_type)
+            plain_self = Instance(info.mro[0], [])  # self-type without type variables
```

Comment:
Hm, `builtins.object` certainly does not contain any type variables, but this may break if we change how attribute access on incompatible selftypes works (#19341 is an example of this problem). If it starts failing more explicitly, binding `self` to `builtins.object` might cause undesired results here. Maybe just get rid of type parameters?

```suggestion
            plain_self = fill_typevars_with_any(info)  # self-type without type variables
```

Reply:
OK, I will try this after we get the mypy-primer.